app/src/main/java/pl/droidevs/books/about/AboutFragment.java

```diff
@@ -0,0 +1,39 @@
+package pl.droidevs.books.about;
+
+import android.arch.lifecycle.ViewModelProviders;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.text.method.LinkMovementMethod;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import pl.droidevs.books.R;
+
+/**
+ * Created by micha on 09.03.2018.
```

Comment:
ten komentarz też można by usunąć, jako że nic nie wnosi. Można ustawić w AS żeby w ogóle go nie dodawał

Reply:
Wybacz 😔
Zapominam usuwać. A nie przyszło mi w ogóle do głowy aby to wyłączyć.